jest.config.js

```diff
@@ -56,9 +56,9 @@ module.exports = {
     `./packages/cactus-test-cmd-api-server/src/test/typescript/integration/remote-plugin-imports.test.ts`,
     `./packages/cactus-test-cmd-api-server/src/test/typescript/integration/plugin-import-with-npm-install.test.ts`,
     `./packages/cactus-test-cmd-api-server/src/test/typescript/integration/plugin-import-with-npm-install-version-selection.test.ts`,
-    `./packages/cactus-test-cmd-api-server/src/test/typescript/integration/runtime-plugin-imports.test.ts`,
     `./packages/cactus-cmd-socketio-server/`,
     `./packages/cactus-plugin-ledger-connector-corda/src/test/typescript/integration/openapi/openapi-validation.test.ts`,
+    `./packages/cactus-cmd-socketio-server/src/main/typescript/routing-interface/util/RIFUtil.test.ts`,
```

Comment:
@awadhana LGTM in general, but wanted to ask why is this test case being added to the ignore list? (looks like it's a jestified test already)